<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server_connection_widget</class>
 <widget class="QMainWindow" name="server_connection_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kiya</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 89, 69);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTip">
    <string>asdf</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tab_widget">
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(77, 89, 69);</string>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="server_tab">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(77, 89, 69);</string>
         </property>
         <attribute name="title">
          <string>Servers</string>
         </attribute>
         <widget class="QTableWidget" name="server_list_view">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>891</width>
            <height>401</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>server</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>project</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>assets</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>shots</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>sequences</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>artists</string>
           </property>
          </column>
         </widget>
         <widget class="QPushButton" name="log_in_button">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>430</y>
            <width>131</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
         <widget class="QLabel" name="log_in_info_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>470</y>
            <width>341</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_line_edit">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>460</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="id_line_edit">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>430</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="log_in_info_label_2">
          <property name="geometry">
           <rect>
            <x>570</x>
            <y>440</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="log_in_info_label_3">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>470</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>PASSWORD</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="scene_tab">
         <attribute name="title">
          <string>Scenes</string>
         </attribute>
         <widget class="QPushButton" name="import_assets_button">
          <property name="geometry">
           <rect>
            <x>770</x>
            <y>440</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Import</string>
          </property>
         </widget>
         <widget class="QPushButton" name="sync_all_assets_button">
          <property name="geometry">
           <rect>
            <x>650</x>
            <y>440</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Sync</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>70</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_all_assets_button">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>130</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="remove_asset_from_import_list_button">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>210</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="remove_all_assets_from_import_list_button">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>270</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
         <widget class="QTableView" name="scene_asset_list_view">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
         <widget class="QTableView" name="assets_to_import_list_view">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>0</y>
            <width>351</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="upload_tab">
         <attribute name="title">
          <string>Update</string>
         </attribute>
         <widget class="QTableView" name="tableView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_2">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>70</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_3">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>130</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_4">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>210</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_5">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>270</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView_2">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>0</y>
            <width>371</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_6">
          <property name="geometry">
           <rect>
            <x>770</x>
            <y>440</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="sync_tab">
         <attribute name="title">
          <string>Sync</string>
         </attribute>
         <widget class="QTableView" name="current_sync_status_list_view">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
         <widget class="QTableView" name="updated_sync_list_view">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>0</y>
            <width>351</width>
            <height>441</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(53, 63, 46);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="add_select_asset_button_7">
          <property name="geometry">
           <rect>
            <x>780</x>
            <y>440</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string>Sync</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
